packages/hash/frontend/src/pages/[account-slug]/index.page.tsx

```diff
@@ -40,11 +41,17 @@ const Page: NextPageWithLayout = () => {
     );
   }
 
-  const ownWorkspace = accountId === authenticatedUser.entityId;
+  const ownWorkspace =
+    accountId ===
+    extractEntityUuidFromEntityId(authenticatedUser.entityEditionId.baseId);
```

Comment:
We probably want to update the little system-type view things (so `AuthenticatedUser`, etc.) to have a `.getId` or something, or `.associatedAccountId`, dunno, but we don't need to pretend these things are entities and would stop us having to call the extract function all over the place, which I am sure we'll forget (or import the incorrect one or something)

Reply:
Changed in e8c2693adbfd9e0c8038e0644dd8e9ea6bbebf34